core-library/src/main/java/org/silverpeas/core/admin/persistence/GroupTable.java

```diff
@@ -157,9 +149,8 @@ public GroupRow getGroup(int superGroupId, String name) throws AdminPersistenceE
     if (groups.size() == 1) {
       return groups.get(0);
     }
-    throw new AdminPersistenceException("GroupTable.getGroup", SilverpeasException.ERROR,
-        "admin.EX_ERR_GROUP_NAME_ID_FOUND_TWICE",
-        "group name: '" + name + "', father group id: '" + superGroupId + "'");
+    throw new AdminPersistenceException(
+        failureOnGetting("subgroup " + name, "in group " + String.valueOf(superGroupId)));
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Directly append the argument of String.valueOf(). [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://integration.silverpeas.org/sonar/coding_rules#rule_key=squid%3AS1153)